CHANGELOG.md

```diff
@@ -1,5 +1,359 @@
+
+  If you've tried to use [Workers KV][kv] in Wrangler, you've probably had a bad time! This PR, along with #334, build support for handling and correctly uploading KV namespace information with your Wrangler project:
+
+  ```
+  [[kv-namespaces]]
+  binding = "NAMESPACE"
+  id = "0f2ac74b498b48028cb68387c421e279"
+  ```
+
+  [ashleymichal]: https://github.com/ashleymichal
+  [pull/333]: https://github.com/cloudflare/wrangler/pull/333
+  [kv]: https://www.cloudflare.com/products/workers-kv/
+
+  - **Adds more descriptive subdomain errors - [EverlastingBugstopper], [issue/207][pull/259]**
+
+  It's super easy to grab a workers.dev subdomain using the `subdomain` command in `wrangler` – so easy, in fact, that many people were trying to use it without even having a Cloudflare account! `wrangler` now warns users when they attempt to add a subdomain without configuring their `account_id` in `wrangler.toml`, as well as when you've already registered a subdomain, or if the subdomain you're trying to register has already been claimed.
+
+  [EverlastingBugstopper]: https://github.com/EverlastingBugstopper
+  [issue/207]: https://github.com/cloudflare/wrangler/issue/207
+  [pull/259]: https://github.com/cloudflare/wrangler/pull/259
+
+  - **Allow custom webpack configuration in wrangler.toml - [EverlastingBugstopper], [issue/246][pull/253]**
+
+  If you'd like to bring your own Webpack config to your Workers project, you can now specify a `webpack_config` key in `wrangler.toml`:
+
+  ```toml
+  webpack_config: webpack.prod.js
+  ```
+
+  [EverlastingBugstopper]: https://github.com/EverlastingBugstopper
+  [issue/246]: https://github.com/cloudflare/wrangler/issue/246
+  [pull/253]: https://github.com/cloudflare/wrangler/pull/253
+
+  - **Enforce one Webpack entry in configuration - [xtuc], [pull/245]**
+
+  `wrangler` now returns an error during the build process if you use a webpack configuration with more than one export – `wrangler` needs to have a single known export from webpack to know what to build!
+
+  [xtuc]: https://github.com/xtuc
+  [pull/245]: https://github.com/cloudflare/wrangler/pull/245
+
+  - **Add user agent - [xtuc], [issue/234][pull/236]**
+
+  For every outgoing request, `wrangler` includes a `User-Agent` header to clearly indicate to servers and APIs that a `wrangler` client is making a request: `wrangler/dev` in debug mode and `wrangler/$version` in release mode.
+
+  [xtuc]: https://github.com/xtuc
+  [issue/234]: https://github.com/cloudflare/wrangler/issue/234
+  [pull/236]: https://github.com/cloudflare/wrangler/pull/236
+
+  - **Display commands in their defined order - [Electroid], [pull/236]**
+
+  We've re-arranged the order of the commands when you run `wrangler` without any subcommands, so that commonly-used commands are easier to find!
+
+  [Electroid]: https://github.com/Electroid
+  [pull/233]: https://github.com/cloudflare/wrangler/pull/236
+
+  - **Show sizes - [xtuc], [pull/205]**
```

Comment:
i'd show an example of what this looks like!

Reply:
good idea!